NEWS.md

```diff
@@ -1,6 +1,13 @@
 
 # renv 0.18.0  (UNRELEASED)
 
+* `dependencies()` now marks `Suggested` packages listed in `DESCRIPTION` files
```

Comment:
What happens if a project contains DESCRIPTION files in sub-folders?

Reply:
That shouldn't have changed — a `DESCRIPTION` in a sub-folder will use `settings$package.dependency.fields`> A `DESCRIPTION` in the project root also uses `Suggests` and `Config/renv/profiles/{profile}/dependencies`.

Reply:
We discussed on Slack, but maybe we can reconsider this part (only look at DESCRIPTION files in the project root)? For a separate PR though, so if this preserves the existing behavior then I think we're happy.